src/framework/statusDropdown/StatusDropdown.jsx

```diff
@@ -56,13 +52,13 @@ export default class StatusDropdown extends Component {
     };
 
     this.optionToStatusMap = {
-      [StatusDropdown.OPTIONS.ACTIVATE]: StatusDot.STATUS.POSITIVE,
-      [StatusDropdown.OPTIONS.DEACTIVATE]: StatusDot.STATUS.NEGATIVE,
+      [StatusDropdown.OPTIONS.ACTIVATE]: DropdownDot.COLOR.GREEN,
```

Comment:
The tests for this component lack the fact that `StatusDropDown.OPTIONS` are properly mapped to corresponding `DropdownDot.COLOR` values. Because there is a mapping involved from status options to the colour values, we should have tests for that here I'd say. 

But in case of `DropDown`, I do not see a mapping involved, that's why I argued there that those tests can be skipped and it is enough to assert that the returned component is a `BaseDropDown`.